VAMobile/src/screens/HealthScreen/Appointments/UpcomingAppointments/UpcomingAppointmentDetails.tsx

```diff
@@ -322,7 +329,7 @@ const UpcomingAppointmentDetails: FC<UpcomingAppointmentDetailsProps> = ({ route
     )
   }
 
-  if (useError(ScreenIDTypesConstants.APPOINTMENTS_SCREEN_ID) || appointmentNotFound) {
+  if (screenError || appointmentNotFound) {
```

Comment:
@dumathane Here's where we added the original logic for seeing if the vetextId fails. Although it's not telling us anything analytics wise, it'll still display the error component to the user when the appointment isn't found